lib/generator.js

```diff
@@ -390,8 +392,13 @@ class Generator {
    * Configure the templates based the desired renderer.
    */
   async configureTemplate() {
-    if (isReactTemplate(this.templateConfig)) {
-      await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
+    if(isReactTemplate(this.templateConfig)) {
+      if(this.compile) {
+        await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
+      } else {
+        // use the cached transpiled version provided by the template
+        this.templateContentDir = path.resolve(this.templateDir, TRANSPILED_TEMPLATE_LOCATION);
+      }
```

Comment:
@derberg is this part correct? I thought when the `this.compile` is set to false, we are supposed to not do the transpilation. But I also don't know exactly what is supposed to be done when `this.compile` is false so I will let you decide : )